pkg/apis/hco/v1beta1/hyperconverged_types.go

```diff
@@ -14,6 +14,33 @@ import (
 // HyperConvergedName is the name of the HyperConverged resource that will be reconciled
 const HyperConvergedName = "kubevirt-hyperconverged"
 
+// HyperConverged is the Schema for the hyperconvergeds API
+// +k8s:openapi-gen=true
+type HyperConvergedConfig struct {
+
+	// nodeSelector is the simplest recommended form of node selection constraint:
+	// It specifies a map of key-value pairs. For the pod to be eligible to run on a node,
```

Comment:
This does not match the body of the object. Description speaks about key-value based node selector, but properties contain matchExpressions which is the elaborate way of selecting nodes.

Reply:
done